### DISCUSSION TITLE
2020/01/22/%E5%BA%9E%E6%B0%8F%E9%AA%97%E5%B1%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#358] [Announcements]
# 2020/01/22/%E5%BA%9E%E6%B0%8F%E9%AA%97%E5%B1%80

一般认为如果在一场金融活动中，投资的主要回报来都自于后加入者的投资资金，就是庞氏骗局。虽然被称为“骗局”，但是庞氏骗局的受害者中，有很多都不是被骗进去的。越是大规模的庞氏骗局，反而越多的人是被贪婪驱动，明知这是一场赌博，依然下场赌一把的。

https://blog.qizhen.xyz/2020/01/22/%E5%BA%9E%E6%B0%8F%E9%AA%97%E5%B1%80

**Comment by ruanqizhen** [comment]
ruanqizhen
2022年08月7日 10:48
比特币刚出现的时候，我曾经想，（[之前的博客](https://blog.qizhen.xyz/2013/09/01/%E6%9C%AA%E6%9D%A5%E7%9A%84%E9%92%B1%E4%BC%9A%E6%98%AF%E4%BB%80%E4%B9%88%E6%A0%B7%E5%AD%90)）如果未来人们都采用类似比特币这样的货币，倒是可以解决政府滥发货币的问题，可以防止通货膨胀。但是后来仔细想想，也许这正是多数国家政府都会禁止此类货币的原因。
历史上，金银铜（而不是纸币），才是世界各国普遍采用的流通货币。采用自然金属作为货币与采用分布式数字货币有相似的地方，就是政府无法滥发货币，毕竟贵金属的蕴藏量和采矿能力都是有限的。但是最终各国政府都不再使用贵金属作为货币了。就是为了能够更有效的控制金融市场。政府怎么可能会再允许一种电子贵金属出来做货币呢。
